.github/workflows/build-and-deploy-pwa.yaml

```diff
@@ -0,0 +1,47 @@
+name: Build and deploy frontend
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  # Build Job
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+      - name: Install Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+      - name: Install Dependencies
+        run: cd power-pay-frontend && npm ci
+
+      - name: Build Project
+        run: cd power-pay-frontend && npm run build
```

Comment:
Why building using `build`, whereas you should be using `build-for-gh`?

Reply:
That is due to my lack of understanding of the syntax. I will change the npm run build to npm run build-for-gh.